chart/inlets-operator/templates/deployment.yaml

```diff
@@ -30,15 +30,13 @@ spec:
         - "-annotated-only"
         {{- end }}
         - "-provider={{.Values.provider}}"
+        - "-zone={{.Values.zone}}"
+        - "-region={{.Values.region}}"
         - "-access-key-file={{.Values.accessKeyFile}}"
         - "-license={{.Values.inletsProLicense}}"
-        - "-region={{.Values.region}}"
-        {{- if .Values.gceProjectId }}
-        - "-zone={{.Values.zone}}"
-        - "-project-id={{.Values.gceProjectId}}"
-        {{- end }}
-        {{- if .Values.packetProjectId }}
-        - "-project-id={{.Values.packetProjectId}}"
+        - "-project-id={{.Values.projectID}}"
```

Comment:
What happens when we set an empty projectID, but it's not required for instance?

Reply:
Sir we simply ignore it as we only use it when we have gce or packet as the exit node providers